tools/src/test/java/org/apache/kafka/tools/MetadataQuorumCommandTest.java

```diff
@@ -73,7 +73,7 @@ public void testDescribeQuorumReplicationSuccessful() throws InterruptedExceptio
         );
 
         List<String> outputs = stream(describeOutput.split("\n")).skip(1).collect(Collectors.toList());
-        if (cluster.config().clusterType() == Type.CO_KRAFT)
+        if (cluster.config().clusterTypes().contains(Type.CO_KRAFT))
```

Comment:
Could you add a new method to expose the `Type`? Otherwise, this check is not accurate since users can set multi-types in `ClusterConfig`